R/Spectra-neutralLoss.R

```diff
@@ -0,0 +1,167 @@
+setClassUnion("functionOrNull", c("function", "NULL"))
+
+#' @title Calculate Neutral Loss Spectra
+#'
+#' @name neutralLoss
+#'
+#' @description
+#'
+#' This help page lists functions to convert MS/MS spectra to neutral loss
+#' spectra. The main function for this is `neutralLoss` and the specific
+#' algorithm to be used is defined (and configured) with dedicated *parameter*
+#' objects (paramer `param` of the `neutralLoss` function.
+#'
+#' The parameter objects for the different algorithms are:
+#'
+#' - `PrecursorMzParam`: calculates neutral loss spectra as in Aisporna 2022 by
+#'   subtracting the (fragment's) peak m/z value from the precursor m/z value
+#'   of each spectrum (precursor m/z - fragment m/z). Parameter `msLevel`
+#'   allows to restrict calculation of neutral loss spectra to specified MS
+#'   level(s). Spectra from other MS level(s) are returned as-is. Parameter
+#'   `filterPeaks` allows in addition to filter fragment peaks e.g. if their
```

Comment:
Not sure what the 'in addtion' means here. If it's in addition to the other parameters, I would drop it. If it's supposed to mean it has multiple effects, then I'm not sure what they are.

Reply:
You're right. Better drop it.